tests/test_create_params.py

```diff
@@ -6,14 +6,14 @@
 def test_create_params_phelps_deaton_with_shocks():
     params = create_params(PHELPS_DEATON_WITH_SHOCKS)
 
-    names = leaf_names(params, separator="$")
+    names = leaf_names(params, separator="__")
     expected_names = [
         "beta",
-        "utility$delta",
-        "next_wealth$interest_rate",
-        "next_wealth$wage",
-        "wage_shock$sd",
-        "additive_utility_shock$scale",
+        "utility__delta",
+        "next_wealth__interest_rate",
+        "next_wealth__wage",
+        "wage_shock__sd",
+        "additive_utility_shock__scale",
```

Comment:
Why did you change the sperator?

Reply:
That was @hmgaudecker's attempt to save his eyes from getting hurt: [268a692](https://github.com/OpenSourceEconomics/lcm/pull/9/commits/268a692041bcfefaab948390d62364151af745c4)

Reply:
:man_shrugging: